FrontDesk/src/BlazorShared/Models/Appointment/CreateAppointmentRequest.cs

```diff
@@ -10,9 +10,7 @@ public class CreateAppointmentRequest : BaseRequest
         public int ClientId { get; set; }
         public int RoomId { get; set; }
         public DateTime DateOfAppointment { get; set; }
-        public TimeSpan Duration { get; set; }
```

Comment:
Don't we need duration?

Reply:
No need as End date calculation in the appointment entity by start and duration from appointment type (UpdateEndTime).